src/styles/Components/src/tokens-menubutton.css

```diff
@@ -0,0 +1,20 @@
+:root {
+    --denhaag-menubutton-active-color: var(--denhaag-menubutton-hover-color);
+    --denhaag-menubutton-active-font-weight: 700;
+
+    --denhaag-menubutton-chevron-active-color: var(--denhaag-menubutton-hover-color);
+    --denhaag-menubutton-chevron-font-size: 12px;
+    --denhaag-menubutton-chevron-hover-color: var(--denhaag-menubutton-chevron--active-color);
```

Comment:
Something messy going on here. The token your referencing is called `..chevron--active-color` but it doesn't exist. (`chevron-active-color` should be correct

Reply:
Fixed